src/App.js

```diff
@@ -100,12 +103,9 @@ export function App() {
   
     const fetchFilteredJobs = useCallback(async () => {
       const url = getUrl(state.page, state.what, state.where, state.distance, state.location0, state.location1, state.location2, state.location3, state.location4, state.location5, state.location6, state.location7, state.category, state.salary_min, state.salary_max, state.full_time, state.part_time, state.company)
-      console.log('url from in fetch ' + url)
       const response = await fetch(url);
       const data = await response.json();
-      console.log('data from fetch:')
-      console.log(data);
-      dispatch({type: 'setJobs', payload: data.results})
+      dispatch({type: 'setJobs', payload: data})
```

Comment:
Yea this will centralize it. I had previously deconstructed it in my component, so I'll keep this change I just need to refactor my component or this will break it

Reply:
done